codesign/bin/codesign.dart

```diff
@@ -14,31 +14,33 @@ import 'package:process/process.dart';
 /// Definitions of variables are included in help texts below.
 const String kHelpFlag = 'help';
 const String kCommitOption = 'commit';
-const String kProductionFlag = 'production';
+const String kDryrunFlag = 'dryrun';
```

Comment:
nit: It seems like this dryrun change could've been a separate PR. This PR is already extra large, so avoid adding in features that are unrelated.

Reply:
umm yeah i agree. umm `dryrun` is actually a rename from the old parameter `production` to make its usage more obvious